fastlane/Fastfile

```diff
@@ -593,6 +593,37 @@ end
       SimCtl.create_device device, devicetype, runtime
     }
   end
+
+  #####################################################################################
+  # UI tests lanes
+  #####################################################################################
+
+    # Props to work done by others in this file and in
+    # https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/fastlane/Fastfile
+    # this is a combined rip-off from both
+
+    #####################################################################################
+    # run_UI_tests
+    # -----------------------------------------------------------------------------------
+    # Usage:
+    # bundle exec fastlane run_UI_tests [device:<Name of the iOS Simulator >]
+    #
+    # Example:
+    # bundle exec fastlane run_UI_tests device:"iPhone 12"
+    #####################################################################################
+  # A lane to run a subset of automated UI tests. Since only login tests are
+  # selected for now, there will be no credentials race condition.
+  desc "Run UI tests"
+  lane :run_UI_tests do | options |
+  scan(
+    workspace: workspace_name,
+    scheme: ui_tests_scheme,
+    device: options[:device],
```

Comment:
TIL: if we don't pass a value when calling the lane, Fastlane will use iPhone 11 as the device value.